File: spring/project2/src/main/java/com/skillstorm/project2/device/Device.java
package com.skillstorm.project2.device;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "device")
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Device {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "number")
    private String number;

    @Column(name = "name")
    private String name;

    @Column(name = "user_id")
    int userId;

    @Override
    public String toString() {
        return "Device{" +
                "id=" + id +
                ", number='" + number + '\'' +
                ", name='" + name + '\'' +
                ", user_id=" + userId +
                '}';
    }
}

File: Project2/project2-angular/src/app/models/device.ts
import User from "./user";

class Device {
    id: number;
    num: number;
    name: string;
    userId: number;
    user: User;

    constructor(id=0, num=0, name="", userId=0, user = new User){
        this.id= id;
        this.num = num;
        this.name = name;
        this.userId = userId;
        this.user = user;
    }
}

export default Device;